custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -76,6 +91,19 @@ def do_predict(self, logger=None):
 
         return response, response_status
 
+    def do_predict(self, logger=None):
+        if self._target_type == TargetType.TRANSFORM:
+            wrong_target_type_error_message = (
+                "This project has target type {}, "
+                "use the /transform/ endpoint.".format(self._target_type)
+            )
+            if logger is not None:
```

Comment:
why would logger be none?

Reply:
not sure- I was following an existing pattern

Reply:
ok I guess we don't have to do this now but still weird.